back/src/main/java/com/qmate/config/SecurityConfig.java

```diff
@@ -64,10 +64,11 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http, ObjectMapper o
             })
         )
         .authorizeHttpRequests(auth -> auth
-            .requestMatchers("/auth/**").permitAll()
-            .requestMatchers(SWAGGER_WHITELIST).permitAll()
-            .requestMatchers("/api/admin/**").hasRole("ADMIN")
-            .anyRequest().authenticated()  // 나머지 모든 요청은 인증 필요
+//            .requestMatchers("/auth/**").permitAll()
+//            .requestMatchers(SWAGGER_WHITELIST).permitAll()
+//            .requestMatchers("/api/admin/**").hasRole("ADMIN")
+//            .anyRequest().authenticated()  // 나머지 모든 요청은 인증 필요
```

Comment:
```suggestion
            .requestMatchers("/auth/**").permitAll()
            .requestMatchers(SWAGGER_WHITELIST).permitAll()
            .requestMatchers("/api/admin/**").hasRole("ADMIN")
            .anyRequest().authenticated()  // 나머지 모든 요청은 인증 필요
```
테스트 하느라 잠깐 주석을 해놓으신거 같아요 주석 해제 부탁드립니다

Reply:
어우 맞아여.. 다시 주석해제 후 원래 상태로 복원해 놓았습니다.